tests/system/filesystem_integrity/data/warning_list.txt

```diff
@@ -0,0 +1,31 @@
+/etc/group
```

Comment:
Add this file to `data/checkfiles/warning_list.txt`. Check that when the framework is installed, this file is placed in this path too

Reply:
Solved https://github.com/wazuh/wazuh-qa/pull/2437/commits/7955e10644d7e0d5c4e2db1e82d28b85e367946a